tsconfig.jest.json

```diff
@@ -5,7 +5,7 @@
     "noImplicitThis": true,
     "moduleResolution": "node",
     "removeComments": false,
-    "target": "es2022",
+    "target": "ESNext",
```

Comment:
we could also rewrite the file so that it inherits the tsconfig.json
https://www.typescriptlang.org/tsconfig/#extends in order to not make this kindof mistake again?

Reply:
Last time I tried to make this file smarter, 10% of the runbots stopped working on spreadsheet and we never found the reason why :sweat_smile:

Reply:
@robodoo r+